ws-client/src/transport.rs

```diff
@@ -227,7 +229,12 @@ impl<'a> WsTransportClientBuilder<'a> {
 					match self.mode {
 						Mode::Plain => TlsOrPlain::Plain(socket),
 						Mode::Tls => {
-							let connector = async_tls::TlsConnector::default();
+							let mut client_config = rustls::ClientConfig::default();
+							if self.use_system_certificates {
+								client_config.root_store = rustls_native_certs::load_native_certs().map_err(|(_, e)| e)?;
```

Comment:
it's fine but you could move this to `fn build` to avoid doing it several times.

Reply:
That would cause the certs to be loaded even when Plain is used. Would something like this 6261d94ed be better?

Reply:
Oh nevermind